python/ray/tune/tests/test_progress_reporter.py

```diff
@@ -203,7 +203,8 @@ def f(config):
 
 EXPECTED_BEST_2 = "Current best trial: 00004 with metric_1=2.0 and parameters={'a': 4}"
 
-EXPECTED_SORT_RESULT_UNSORTED = """Number of trials: 5 (1 PENDING, 1 RUNNING, 3 TERMINATED)
+EXPECTED_SORT_RESULT_UNSORTED = """Number of trials: 5 (1 PENDING, 1 RUNNING, \
+3 TERMINATED)
```

Comment:
nit: Add the backslash directly after the three double quotes because the message can be put in single line in this way, which can improve readability.